course05/meeting00/CMakeLists.txt

```diff
@@ -1,4 +1,15 @@
 cmake_minimum_required(VERSION 3.1)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -g -Wall")
+# Would you normally default CMAKE_EXPORT_COMPILE_COMMANDS to on? And is this the way to do it?
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# Would you normally set the C++ standard in this way? I can imagine it can help CMake to know it's compiling for C++14?
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
You are right, this is definitely the way to do it - my CMakeLists.txt is not portable!